draft-ietf-quic-qpack.md

```diff
@@ -272,25 +270,26 @@ An encoder can decide whether to risk having a stream become blocked. If
 permitted by the value of SETTINGS_QPACK_BLOCKED_STREAMS, compression efficiency
 can often be improved by referencing dynamic table entries that are still in
 transit, but if there is loss or reordering the stream can become blocked at the
-decoder.  An encoder avoids the risk of blocking by only referencing dynamic
+decoder.  An encoder can avoid the risk of blocking by only referencing dynamic
 table entries which have been acknowledged, but this could mean using
 literals. Since literals make the header block larger, this can result in the
 encoder becoming blocked on congestion or flow control limits.
 
 ### Known Received Count
 
-In order to identify which dynamic table entries can be safely used without a
-stream becoming blocked, the encoder tracks the number of entries received by
-the decoder.  The Known Received Count tracks the total number of acknowledged
-insertions.
+The Known Received Count is the total number of dynamic table insertions and
+duplications acknowledged by the decoder.  The encoder tracks the Known Received
+Count in order to identify which dynamic table entries can be references without
+potentially blocking a stream.  The decoder tracks the Known Received Count in
+order to be able to send Insert Count Increment instructions (see
+{{insert-count-increment}}).
 
-When blocking references are permitted, the encoder uses Header Acknowledgement
-instructions ({{header-acknowledgement}}) to maintain the Known Received
-Count.  If a header block was potentially blocking, the acknowledgement implies
-that the decoder has received all dynamic table state necessary to process the
-header block.  If the Required Insert Count of an acknowledged header block was
-greater than the encoder's current Known Received Count, the block's Required
-Insert Count becomes the new Known Received Count.
```

Comment:
"the block's Required Insert Count becomes the new Known Received Count" always trips me: does the value of the Required Insert Count change to the value of the Known Received Count, or does the value of the Required Insert Count became to be known as the new value of the Known Received Count?